dti-website-redesign/src/components/Footer.tsx

```diff
@@ -29,10 +26,40 @@ const icons: Icon[] = [
   }
 ];
 
+const EmailIcon: React.FC = () => {
+  const [isNotificationVisible, setIsNotificationVisible] = useState(false);
+  const target = useRef(null);
+  return (
+    <>
+      <img
+        className={styles.socialIcon}
+        src={'/static/icons/email.svg'}
+        alt={'email icon'}
+        ref={target}
+        onClick={() => {
+          navigator.clipboard.writeText('hello@cornelldti.org');
+          setIsNotificationVisible(true);
+        }}
+      />
+      <Overlay placement={'top'} show={isNotificationVisible} target={target.current}>
+        <div style={{ position: 'absolute' }} id={`tooltip-top`}>
```

Comment:
I dont think we need this id anymore
```suggestion
        <div style={{ position: 'absolute' }}>
```